vital/algo/estimate_similarity_transform.h

```diff
@@ -114,19 +130,21 @@ class VITAL_ALGO_EXPORT estimate_similarity_transform
    *          \c from space to camera centers in the \c to space.
    */
   virtual kwiver::vital::similarity_d
-  estimate_transform(kwiver::vital::landmark_map_sptr const from,
-                     kwiver::vital::landmark_map_sptr const to) const;
+  estimate_transform( kwiver::vital::landmark_map_sptr const from,
+                      kwiver::vital::landmark_map_sptr const to ) const;
 
 protected:
-    estimate_similarity_transform();
-
+  estimate_similarity_transform();
 };
 
 /// Shared pointer for similarity transformation algorithms
-typedef std::shared_ptr<estimate_similarity_transform> estimate_similarity_transform_sptr;
+typedef std::shared_ptr< estimate_similarity_transform >
+  estimate_similarity_transform_sptr;
 
 } // end namespace algo
```

Comment:
these comments can just be // namespace for consistency 
```suggestion
} // namespace algo
```